docs/config.md

```diff
@@ -127,38 +127,42 @@ for example `TRACES`, and `{0}` is the case-sensitive name of the instrumentatio
+| `QUARTZ`              | [Quartz](https://www.nuget.org/packages/Quartz) **Not supported on .NET Framework 4.7.1 and older**                                                                                                                | ≥3.4.0                 | source               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `SQLCLIENT`           | [Microsoft.Data.SqlClient](https://www.nuget.org/packages/Microsoft.Data.SqlClient), [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient) and `System.Data` (shipped with .NET Framework) | * \[6\]                | source               | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `STACKEXCHANGEREDIS`  | [StackExchange.Redis](https://www.nuget.org/packages/StackExchange.Redis) **Not supported on .NET Framework**                                                                                                      | ≥2.0.405 & < 3.0.0     | source & bytecode    | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `WCFCLIENT`           | WCF                                                                                                                                                                                                                | *                      | source & bytecode    | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
+| `WCFSERVICE`          | WCF **Not supported on .NET**.                                                                                                                                                                                     | *                      | source & bytecode    | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
 
 \[1\]: Only integrated pipeline mode is supported.
 
-\[2\]: `Azure.` prefixed packages, released after October 1, 2021.
+\[2\]: `ASP.NET (.NET Framework) MVC / WebApi` is not supported on ARM64.
 
-\[3\]: `Elastic.Clients.Elasticsearch` version ≥8.0.0 and <8.10.0.
+\[3\]: `Azure.` prefixed packages, released after October 1, 2021.
+
+\[4\]: `Elastic.Clients.Elasticsearch` version ≥8.0.0 and <8.10.0.
         Version ≥8.10.0 is supported by `Elastic.Transport` instrumentation.
 
-\[4\]: `Microsoft.Data.SqlClient` v3.* is not supported on .NET Framework,
+\[5\]: `Confluent.Kafka` is supported from version ≥1.6.2 on ARM64.
```

Comment:
Should we just say that it is supported from version >= 1.8.2 on ARM64 because of the problems that you noticed with the tests?

Reply:
1.6.3 and 1.7.0 seem to be one of the top downloaded versions. Unfortunately I don't know how important are these versions for arm64. In the same time 1.8.0 and 1.8.1 are one of the least downloaded versions plus there is 1.8.2 also extremely popular. So we may just add that 1.8.0-1.8.1 are not guaranteed to work.

Reply:
I am fine with either calling out that we support 1.8.2+, or saying that we support >= 1.6.2 with the caveat of not supporting 1.8.0 and 1.81 (supporting 1.8.2+ is easier to understand). I just think that it is misleading to say that we support 1.6.2+ when we were not able to validate 1.8.0 and 1.8.1 (we do not understand why it did not work yet).

Reply:
Agree, let's start with 1.8.2+. If anyone feels it should be lower, then they can always open an issue.